backend/app/adapter/db/url_integration_test.go

```diff
@@ -242,6 +242,117 @@ func TestURLSql_Create(t *testing.T) {
 	}
 }
 
+func TestURLSql_GetByAliases(t *testing.T) {
+	twoYearsAgo := mustParseTime(t, "2017-05-01T08:02:16-07:00")
+	now := mustParseTime(t, "2019-05-01T08:02:16-07:00")
+
+	testCases := []struct {
+		name         string
+		tableRows    []urlTableRow
+		aliases      []string
+		hasErr       bool
+		expectedURLs []entity.URL
+	}{
+		{
+			name:      "alias not found",
+			tableRows: []urlTableRow{},
+			aliases:   []string{"220uFicCJj"},
+			hasErr:    true,
+		},
+		{
+			name: "found url",
+			tableRows: []urlTableRow{
+				{
+					alias:     "220uFicCJj",
+					longLink:  "http://www.google.com",
+					createdAt: &twoYearsAgo,
+					expireAt:  &now,
+					updatedAt: &now,
+				},
+				{
+					alias:     "yDOBcj5HIPbUAsw",
+					longLink:  "http://www.facebook.com",
+					createdAt: &twoYearsAgo,
+					expireAt:  &now,
+					updatedAt: &now,
+				},
+			},
+			aliases: []string{"220uFicCJj", "yDOBcj5HIPbUAsw"},
+			hasErr:  false,
+			expectedURLs: []entity.URL{
+				entity.URL{
+					Alias:       "220uFicCJj",
+					OriginalURL: "http://www.google.com",
+					CreatedAt:   &twoYearsAgo,
+					ExpireAt:    &now,
+					UpdatedAt:   &now,
+				},
+				entity.URL{
+					Alias:       "yDOBcj5HIPbUAsw",
+					OriginalURL: "http://www.facebook.com",
+					CreatedAt:   &twoYearsAgo,
+					ExpireAt:    &now,
+					UpdatedAt:   &now,
+				},
+			},
+		},
+		{
+			name: "nil time",
+			tableRows: []urlTableRow{
+				{
+					alias:     "220uFicCJj",
+					longLink:  "http://www.google.com",
+					createdAt: nil,
+					expireAt:  nil,
+					updatedAt: nil,
+				},
+				{
+					alias:     "yDOBcj5HIPbUAsw",
+					longLink:  "http://www.facebook.com",
+					createdAt: &twoYearsAgo,
+					expireAt:  &now,
+					updatedAt: &now,
+				},
+			},
+			aliases: []string{"220uFicCJj"},
+			hasErr:  false,
+			expectedURLs: []entity.URL{
+				entity.URL{
+					Alias:       "220uFicCJj",
+					OriginalURL: "http://www.google.com",
+					CreatedAt:   nil,
+					ExpireAt:    nil,
+					UpdatedAt:   nil,
+				},
+			},
+		},
```

Comment:
I think this test case is unnecessary. Are you trying to cover specific errors?

Reply:
trying to test urls with different time field, but you are right, I just removed it